UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/ChunkedTileMap.cs

```diff
@@ -279,6 +342,121 @@ IEnumerator IEnumerable.GetEnumerator()
 		return GetEnumerator();
 	}
 
+	public Vector3Int[,] GetAllPositions()
```

Comment:
why do you need this, if you're using this in the pathfinding it will be very very slow

Reply:
I got tired of trying to get paths within specific chunks (didn't work well), so I just yoink all positions now.

Reply:
wait, nvm
this is remnants of an old method before `FromTo`, it is not being used by anything at the moment.

Reply:
![image](https://github.com/user-attachments/assets/bb7f1a58-4110-4187-8e00-b9989fd78d65)

Reply:
I would just say change the method name to highlight it is slow

Reply:
Done